libs/net/kafka/Models/NlpRequest.cs

```diff
@@ -0,0 +1,42 @@
+using TNO.API.Areas.Services.Models.Content;
+
+namespace TNO.Kafka.Models;
+
+/// <summary>
+/// NLPRequest class, provides a model for requesting natural language processing.
+/// TODO: Change name to NLPRequestModel for consistent naming convention
+/// </summary>
+public class NLPRequest
+{
+    #region Properties
+    /// <summary>
+    /// get/set - The primary key to identify the content to index.
+    /// </summary>
+    public long ContentId { get; set; }
```

Comment:
Right now this is all we need for the NLP Service to know what to do.  At some point I'd love to not link the services so tightly, but time and budget require compromise.